### DISCUSSION TITLE
Load RDF/Turtle data

### DISCUSSION THREAD

**Discussion by kesavkolla** [apache/incubator-hugegraph#2145] [Q&A] [answered]
Is it possible to load data in RDF/Turtle triplet format into hugegraph? I have data in turtle format and I want to do bulk ingest of that data into hugegraph. Looking for ways on how to do it.

Also is it possible to integrate hugegraph with spark GraphX?

**Comment by imbajin** [comment, marked as answer]
Currently, we need transfer data first, like `rdf --> csv/json`, then use `hugegraph-toolchain`'s loader component to load data:
-  [convert-rdf-file-to-csv-file-without-programming.html](https://www.withdata.com/blog/datafileconverter/convert-rdf-file-to-csv-file-without-programming.html) (like this way)
- [online RDF-CSV transfer](https://sendeyo.com/onlineconverter/en/rdf/text-csv)
- .... (like search in github repo)

>  Also is it possible to integrate hugegraph with spark GraphX?

Yep, we support it in the early version(U could find it in the historical link), but now we recommend to use our separate **graph computing** component ([hugegraph-computer](https://github.com/apache/incubator-hugegraph-computer/)), and we also support load data with Spark/Flink in `huge-loader` now (in toolchain)
